TheBereftSouls.cs

```diff
@@ -6,13 +6,34 @@ public class TheBereftSouls : Mod
 {
     internal static TheBereftSouls Instance = null!;
```

Comment:
make this `{get; private set;}`

Reply:
how would you pair this with TryGetMod? changing all of the instances to get; private set; errors TryGetMod with "A non ref-returning property or indexer may not be used as an out or ref value"

Reply:
right, properties do that. can't pass them as ref or out cuz they're functions (which get inlined but that's not relevant).

you could either keep the code as-is, or add private members which the properties get and set, then pass the member as an out param. the first is probably better.